GLAA.Web/Views/PublicRegister/Index.cshtml

```diff
@@ -62,8 +62,8 @@
                     <td>@licence.ApplicationId</td>
                     <td>@licence.BusinessName</td>
                     <td>@licence.TradingName</td>
-                    <td>@licence.Country</td>
-                    <td>@licence.County</td>
+                    <td>@(Model.Countries?.SingleOrDefault(x => int.Parse(x.Value) == licence.CountryId)?.Text ?? string.Empty)</td>
```

Comment:
At some point I think we should just store the country as an object not just an FK, then we can do away with the need to hook into the list here